flight/Modules/RadioComBridge/RadioComBridge.c

```diff
@@ -379,10 +378,10 @@ static void radioRxTask( __attribute__ ((unused))
 #ifdef PIOS_INCLUDE_WDG
 		PIOS_WDG_UpdateFlag(PIOS_WDG_RADIORX);
 #endif
-		if (PIOS_COM_RADIO) {
+		if (PIOS_COM_RFM22B) {
```

Comment:
The problem is no one calls the module init methods.  It's done from putting pointers in an initcall section automatically, so there's not really a way to say what port you mean.

There's other ways but they use a few bytes of ram, and probably aren't worth it